libopeniscsiusr/idbm.c

```diff
@@ -447,8 +447,6 @@ static int _idbm_iface_rec_link(struct iscsi_iface *iface,
 			 IDBM_SHOW, num, _CAN_MODIFY);
 		_rec_str(IFACE_DHCP_VID_STR, recs, iface, dhcp_vendor_id,
 			 IDBM_SHOW, num, _CAN_MODIFY);
-		_rec_str(IFACE_DHCP_SLP_DA, recs, iface, dhcp_slp_da, IDBM_SHOW,
-			 num, _CAN_MODIFY);
```

Comment:
What was the reason for removing this one? Is the idea that it doesn't work or maybe not used like how the open-iscsi/usr might be?

Reply:
No, just an error on my part. I will fix this.

Since there are two fixes now to my submission, I will just fix them both and force-push the update.